spec.html

```diff
@@ -112,3 +119,123 @@
   <p>A conforming implementation, when processing strict mode code, must not extend, as described in <emu-xref href="#sec-additional-syntax-numeric-literals"></emu-xref>, the syntax of |NumericLiteral| to include |LegacyOctalIntegerLiteral|, nor extend the syntax of |DecimalIntegerLiteral| to include |NonOctalDecimalIntegerLiteral|.</p>
 
 </emu-clause>
+
+<!-- es6num="7.1.3.1" -->
+<emu-clause id="sec-tonumber-applied-to-the-string-type">
+  <h1>ToNumber Applied to the String Type</h1>
+  <p>ToNumber applied to Strings applies the following grammar to the input String interpreted as a sequence of UTF-16 encoded code points (<emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>). If the grammar cannot interpret the String as an expansion of |StringNumericLiteral|, then the result of ToNumber is *NaN*.</p>
+  <emu-note>
+    <p>The terminal symbols of this grammar are all composed of Unicode BMP code points so the result will be *NaN* if the string contains the UTF-16 encoding of any supplementary code points or any unpaired surrogate code points.</p>
+  </emu-note>
+  <h2>Syntax</h2>
+  <emu-grammar>
+    StringNumericLiteral :::
```

Comment:
For consistency,is there a reason we should call this StringNumericLiteral as opposed to StrNumericLiteral?